en/Plugins/Tags view.md

```diff
@@ -2,7 +2,7 @@
 aliases: [Tag pane]
```

Comment:
Can you add `- Plugins/Tags` as an alias just so we can attempt to catch any old links shared out there?

Reply:
I've added it. (I'm not so familiar with the OSS workflows, but I allowed edits by maintainers, so I thought you could do it without me.)

Reply:
Particularly, I want to preserve you as being the one who made the changes. For posterity. Some people care about that. o.o;

Reply:
I see, I've learned something new. Thank you!